src/helpers/accountId.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+type Params = {
+  type: string,
+  xpub: string,
+  walletName: string,
+}
+
+export function encode({ type, xpub, walletName }: Params) {
+  return `${type}:${xpub}:${walletName}`
+}
+
+export function decode(accountId: string) {
+  const [type, xpub, walletName] = accountId.split(':')
```

Comment:
please make this function returning a `Params`. & also typecheck things (so it's actually all typeof string) and throw if it's not valid. because in future if this don't decode, like we have old id, etc.. maybe it's better to have more clear validation.
maybe you could even have a `libcore1` suffix just in case we want to do some logic in future or change the format? 🤔